codemods/1.0.0/package-renames.ts

```diff
@@ -8,9 +8,10 @@ const renameMap: { [key: string]: string } = {
     "@vivid-planet/react-admin-form": "@comet/admin",
     "@vivid-planet/react-admin-layout": "@comet/admin",
     "@vivid-planet/react-select": "@comet/admin-react-select",
-    "@vivid-planet/react-admin-date-picker": "@comet/admin-date-picker",
+    "@vivid-planet/react-admin-date-picker": "@comet/admin-date-time",
     "@vivid-planet/react-admin-color-picker": "@comet/admin-color-picker",
     "@vivid-planet/react-admin-rte": "@comet/admin-rte",
+    "@comet/admin-date-picker": "@comet/admin-date-time",
```

Comment:
you have to create a new codemod for 2.0.0

Reply:
I added codemods for 2.0.0